rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -848,6 +849,7 @@ protected NodeRunner buildNodeRunner() {
                     getRepositoryLocator()
             ));
         }
+        Secp256k1.initialize(getRskSystemProperties());
```

Comment:
I would move this somewhere else because CLI tools don't build the node runner and thus they never execute this line. We would want ideally to have this run in those tools as well.

Reply:
I've been thinking about that... Where else would you suggest to put it?
Because if you dont initialize, it would take default Bouncy Castle. So CLI tools not taking the config... shouldn’t be actually a problem, plus CLI are still experimental.
And couldn't find any other possible way of running (besides CLI) without building a NodeRunner, Fed Node extends from this, so its safe.